common/styles/utilities/_root-scope-classes.scss

```diff
@@ -454,3 +454,10 @@
 .width-auto {
   width: auto;
 }
+
+.base-heading-highlight {
```

Comment:
I feel a bit uneasy about having this in the `root-scope-classes` – perhaps having a dedicated `base_heading.scss` partial with only this in would be less weird (because realistically this is too bespoke to be used anywhere else).

Alternatively it could go in a `<style>` block in the `BaseHeading` component, but we'd lose the benefit of having auto-prefixer adding the browser prefixes still required for `box-decoration-break`.

Thoughts?

Reply:
Could we add it to the style of the `Heading` and have some fancy JSX prefix it for us?
e.g. https://github.com/rofrischmann/inline-style-prefixer

Reply:
Maybe. Or write the extra line of css inline ourselves.

Reply:
> Maybe. Or write the extra line of css inline ourselves.

👍

Reply:
I came down in favour of having a `_highlighted_heading.scss` partial, to keep things sane.